package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@boringcodes/backpack",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Don't change the version here. I will run "yarn release" and "standard-version" will do it along with updating CHANGELOG file for us